scripts/exregional_refl2tten.sh

```diff
@@ -171,19 +171,36 @@ else
     bkpath=${cycle_dir}${slash_ensmem_subdir}/fcst_fv3lam${cycle_tag}/INPUT
 fi
 
+n_iolayouty=$(($IO_LAYOUT_Y-1))
+list_iolayout=$(seq 0 $n_iolayouty)
+
 cp_vrfy ${fixgriddir}/fv3_akbk                               fv3_akbk
 cp_vrfy ${fixgriddir}/fv3_grid_spec                          fv3_grid_spec
 
-if [ -r ${bkpath}/phy_data.nc ]; then  # Use background from warm restart
-  ln_vrfy -s ${bkpath}/fv_core.res.tile1.nc         fv3_dynvars
-  ln_vrfy -s ${bkpath}/fv_tracer.res.tile1.nc       fv3_tracer
-  ln_vrfy -s ${bkpath}/sfc_data.nc                  fv3_sfcdata
-  ln_vrfy -s ${bkpath}/phy_data.nc                  fv3_phydata
+if [ -r "${bkpath}/coupler.res" ]; then # Use background from warm restart
+  if [ "${IO_LAYOUT_Y}" == "1" ]; then
+    ln_vrfy -s ${bkpath}/fv_core.res.tile1.nc         fv3_dynvars
+    ln_vrfy -s ${bkpath}/fv_tracer.res.tile1.nc       fv3_tracer
+    ln_vrfy -s ${bkpath}/sfc_data.nc                  fv3_sfcdata
+    ln_vrfy -s ${bkpath}/phy_data.nc                  fv3_phydata
+  else
+    for ii in ${list_iolayout}
+    do
+      iii=`printf %4.4i $ii`
```

Comment:
suggest to use `iii=$(printf %4.4i $ii)`

Reply:
done